src/utils.js

```diff
@@ -348,6 +348,9 @@ function splitElements(raw) {
 */
 function getScope(data, elements=[]) {
 	for (let elt of elements) {
+		if (elt == '__proto__')
```

Comment:
It would be better adding `prototype` and `constructor` to the blacklist :smile:

Reply:
I've updated as you suggested. Also used a clean approach without `if` statement